src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -89,6 +89,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1009 = 1009,
 
+        /// <summary>
+        /// Unable to parse project information.
+        /// </summary>
+        NU1010 = 1010,
```

Comment:
Why not just re-use https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1105/ ? 

They communicate the same thing.

Reply:
Because I didn't spend enough time looking up existing error codes. Thanks for letting me know about it.